src/utils/wallet.js

```diff
@@ -527,7 +527,8 @@ class Wallet {
         const balance = await account.getAccountBalance()
 
         // TODO: Should lockup contract balance be retrieved separately only when needed?
-        const lockupAccountId = accountId + '.' + LOCKUP_ACCOUNT_ID_SUFFIX
+        const lockupAccountId = accountId.replace('.' + ACCOUNT_ID_SUFFIX, '') + '.' + LOCKUP_ACCOUNT_ID_SUFFIX
```

Comment:
this will fail for e.g. `testnet.is.testnet` account

Reply:
I suppose I'd call that an edge case, though it's a fair point.

Reply:
Seems like there could be a few edge cases here, which is the issue with not using the entire account name (including the account suffix) in the lockup schema.

Reply:
I think it's relatively straighforward if we actually replace only suffix, i.e. smth like `accountId.replace(/\.testnet$/, '.' + LOCKUP_ACCOUNT_ID_SUFFIX)` would work